### ISSUE TITLE
Upgrade and downgrade support

### ISSUE THREAD

**Issue by zamazan4ik** [chaos-mesh/chaos-mesh#3575]
Hi.

After reading the documentation for me are not clear the following questions:
- Could I update Chaos Mesh/chaosd without downtime? I mean, during upgrade to the newer version, all ongoing experiments will work fine, dashboard will work, etc. The same question is related to downgrade
- I do not see "Monitoring" part in the documentation. Not sure, maybe there are some metrics, which could be monitored with e.g. Prometheus. And maybe for some metrics will be a good idea to configure some alerts.

Thanks in advance!

**Comment by STRRL** [comment]
> Could I update Chaos Mesh/chaosd without downtime? I mean, during upgrade to the newer version, all ongoing experiments will work fine, dashboard will work, etc. The same question is related to downgrade

No. Chaos Mesh could not upgrade/downgrade with zero downtime now. 😢 We suggest that upgrading Chaos Mesh without executing experiments. And we are working on some features like chaos-daemon graceful-shutdown and recover experiment after the restart. I think it could be helpful for this feature.

> I do not see "Monitoring" part in the documentation. Not sure, maybe there are some metrics, which could be monitored with e.g. Prometheus. And maybe for some metrics will be a good idea to configure some alerts.

Here are some examples of monitoring: https://github.com/chaos-mesh/chaos-mesh/tree/master/manifests/monitoring. I think we could move them into documentation website.

**Comment by zamazan4ik** [comment]
Could you please put this information into the documentation? Thanks!
